<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸识别考勤系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.837, y1:0.528682, x2:0.567, y2:0.465682, stop:0.0995025 rgba(200, 224, 239, 255), stop:0.965174 rgba(230, 240, 240, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_cricle">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:150px;
border-image: url(:/icon/face3.png);
background-color:rgb(197, 224, 239,50);






</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_cricle_big">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:300px;
background-image:url()0;
background-color: rgb(0, 50, 90);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5,
stop:0 rgba(255, 255, 255, 250), 
stop:0.3 rgba(255, 255, 255, 80), 
stop:0.4 rgba(255, 255, 255, 80), 
stop:0.425 rgba(255, 255, 255, 200), 
stop:0.5 rgba(255, 255, 255, 0), 
stop:1 rgba(255, 255, 255, 0)
);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_top_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255,200);
border-radius:15px;
background-color: qlineargradient(spread:reflect, x1:0.837, y1:0.528682, x2:0.567, y2:0.465682, stop:0.0995025 rgba(217, 233, 246,150), stop:0.965174 rgba(228, 240, 248,150));


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>450</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/refresh.png);
background-color:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>571</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addface_btn">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{  
border:3px solid rgb(141, 255, 241,200);
border-radius:15px;
background-color:rgb(195, 224, 239);
}
QPushButton:pressed{  
border:3px solid rgb(141, 255, 241);
background-color: rgb(141, 255, 241);
border-radius:15px;
color:rgb(255,255,255);
}
</string>
       </property>
       <property name="text">
        <string>录入人脸</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recface_btn">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{  
border:3px solid rgb(141, 255, 241,200);
border-radius:15px;
background-color:rgb(195, 224, 239);
}
QPushButton:pressed{  
border:3px solid rgb(141, 255, 241);
background-color: rgb(141, 255, 241);
border-radius:15px;
color:rgb(255,255,255);
}
</string>
       </property>
       <property name="text">
        <string>人脸识别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enter_btn">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{  
border:3px solid rgb(141, 255, 241,200);
border-radius:15px;
background-color:rgb(195, 224, 239);
}
QPushButton:pressed{  
border:3px solid rgb(141, 255, 241);
background-color: rgb(141, 255, 241);
border-radius:15px;
color:rgb(255,255,255);
}
</string>
       </property>
       <property name="text">
        <string>进入系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_month">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 123, 204,200);
border-radius:5px;
background-color:none;
font-weight:300;
font: 20pt;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>34</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 123, 204,180);
border-radius:5px;
background-color:none;
font-weight:300;
font: 18pt;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wait">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>345</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color:rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>70</y>
      <width>97</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font:14px;
border:3px solid rgb(141, 255, 241,200);
border-radius:5px;
background-color:rgb(195, 224, 239);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>活体置信度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="conf_fake_tip">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 170, 127);
font:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>口罩识别</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="conf_mask_tip">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 170, 127);
font:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>识别结果</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name_tip">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 170, 127);
font:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>人脸相似度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="conf_face_tip">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 170, 127);
font:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label_cricle_big</zorder>
   <zorder>label_top_2</zorder>
   <zorder>refresh</zorder>
   <zorder>horizontalWidget</zorder>
   <zorder>label_month</zorder>
   <zorder>label_time</zorder>
   <zorder>label_cricle</zorder>
   <zorder>label_wait</zorder>
   <zorder>verticalFrame</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
